contracts/stellar-interchain-token/src/contract.rs

```diff
@@ -29,6 +29,8 @@ impl InterchainToken {
     ) {
         interfaces::set_owner(&env, &owner);
 
+        storage::set_token_manager(&env, &owner);
```

Comment:
Why are we changing the interchain token contract again? That requires migrating every instance again. The owner can be kept as ITS as we discussed in a previous PR. The token manager is already an additional minter

Reply:
The point is that the interchain token isn't meant to be used for custom token linking. That's meant for other custom logic, they just need to implement the mint interface and allow the token manager permission to call it